lighthouse/tests/beacon_node.rs

```diff
@@ -2505,13 +2518,13 @@ fn logfile_format_flag() {
 fn sync_eth1_chain_default() {
     CommandLineTest::new()
         .run_with_zero_port()
-        .with_config(|config| assert_eq!(config.sync_eth1_chain, false));
+        .with_config(|config| assert_eq!(config.sync_eth1_chain, true));
```

Comment:
Note that `sync_eth1_chain` will now effectively default to `true` since it is set to `true` when `--execution_endpoint` is provided (which will now be all the time).

This may very well deprecate this config value entirely, although it can still technically be set back to `false` using the `--disable-deposit-contract-sync` flag.